### ISSUE TITLE
6.4.0 assets:precompile fails with "The `date-selection` class does not exist"

### ISSUE THREAD

**Issue by neubi4** [zammad/zammad#5394]
### Used Zammad Version

6.4.0

### Environment

- Installation method: source
- Operating system (if you're unsure: `cat /etc/os-release` ): Debian 12
- Database + version: postgresql 15
- Elasticsearch version: 7.10.2
- Browser + version: Firefox 128.e ESR

### Actual behaviour

When Updating our Source Installation to 6.4.0, rake assets:precompile fails

```
$ rake assets:precompile
Lockfile is up to date, resolution step is skipped
Packages: +1
+
Progress: resolved 1, reused 1, downloaded 0, added 1, done
Done in 2s
Building with Vite ⚡️
vite v5.4.8 building for production...
transforming...
✓ 2203 modules transformed.
x Build failed in 9.94s
error during build:
[vite-plugin-pwa:build] [postcss] /opt/zammad/app/frontend/shared/components/Form/fields/FieldDate/FieldDateTimeInput.vue?vue&type=style&index=0&lang.css:28:3: The `date-selection` class does not exist. If `date-selection` is a custom class, make sure it is defined within a `@layer` directive.
file: /opt/zammad/app/frontend/shared/components/Form/fields/FieldDate/FieldDateTimeInput.vue?vue&type=style&index=0&lang.css:28:2
    at Input.error (/opt/zammad/node_modules/.pnpm/postcss@8.4.47/node_modules/postcss/lib/input.js:106:16)
    at AtRule.error (/opt/zammad/node_modules/.pnpm/postcss@8.4.47/node_modules/postcss/lib/node.js:115:32)
    at processApply (/opt/zammad/node_modules/.pnpm/tailwindcss@3.4.13/node_modules/tailwindcss/lib/lib/expandApplyAtRules.js:380:29)
    at /opt/zammad/node_modules/.pnpm/tailwindcss@3.4.13/node_modules/tailwindcss/lib/lib/expandApplyAtRules.js:551:9
    at /opt/zammad/node_modules/.pnpm/tailwindcss@3.4.13/node_modules/tailwindcss/lib/processTailwindFeatures.js:55:50
    at async Array.plugins (/opt/zammad/node_modules/.pnpm/tailwindcss@3.4.13/node_modules/tailwindcss/lib/plugin.js:38:17)
    at async LazyResult.runAsync (/opt/zammad/node_modules/.pnpm/postcss@8.4.47/node_modules/postcss/lib/lazy-result.js:261:11)
    at async compileCSS (file:///opt/zammad/node_modules/.pnpm/vite@5.4.8_@types+node@18.11.4_sass@1.79.4_terser@5.31.0/node_modules/vite/dist/node/chunks/dep-CDnG8rE7.js:36860:21)
    at async Object.transform (file:///opt/zammad/node_modules/.pnpm/vite@5.4.8_@types+node@18.11.4_sass@1.79.4_terser@5.31.0/node_modules/vite/dist/node/chunks/dep-CDnG8rE7.js:36141:11)
    at async transform (file:///opt/zammad/node_modules/.pnpm/rollup@4.21.0/node_modules/rollup/dist/es/shared/node-entry.js:18808:16)
    at async ModuleLoader.addModuleSource (file:///opt/zammad/node_modules/.pnpm/rollup@4.21.0/node_modules/rollup/dist/es/shared/node-entry.js:19024:36)

pid 914423 exit 1
Build with Vite failed! ❌
```

We cleared the cache with `rails r "Rails.cache.clear"` before running the precompile command

### Expected behaviour

rake assets:precompile is successfull

### Steps to reproduce the behaviour

nodejs v20.18.0
yarn 1.22.22
pnpm 9.12.3
ruby 3.2.4 (2024-04-23 revision af471c0e01) [x86_64-linux]

### I'm sure this is a bug and no feature request or a general question.

yes

**Comment by dvuckovic** [comment]
The offending file shown in the log (`app/frontend/shared/components/Form/fields/FieldDate/FieldDateTimeInput.vue`) is not part of the Zammad codebase anymore. I wonder why do you still have it in your source tree? 🤔

Perhaps you can check how you updated to the new version, and why do you have this extra file in your filesystem.

**Comment by dominikklein** [comment]
I think in the past this problem was coming from an wrong update process, because of that old/no longer existing files were still present in the code base.

**Comment by neubi4** [comment]
You are correct, after removing the whole `/opt/zammad` directory and basically beginning from zero, precompile works.

This is our first upgrade since we just started using zammad. The update docs for source installations states to just unpack the new release over the existing folder: https://docs.zammad.org/en/latest/install/update.html

Deleting and recreating the `/opt/zammad` directory can be problematic because this is the home dir of the zammad user and contains other files like the modified `.bashrc`.

This is not a problem for us since we automated creating of the user and installing zammad with ansible, so we can remove the dir every time we update. But maybe there are some manual source installations where doing this is not an option.

By looking at the update workflow i currently have no better idea. Maybe there should be an warning or an info about this problem in the update docs?
